docs/source/beta.rst

```diff
@@ -270,27 +285,38 @@ To log in to the qpc command line interface, enter the following command:
 
 The ``qpc server login`` command retrieves a token that is used for authentication with subsequent command line interface commands. That token is removed when you log out of the server. To log out of the server, enter the following command:
 
-``qpc server logout``
+  ``# qpc server logout``
 
 Getting Started with Quipucords
 -------------------------------
-You use the capabilities of Quipucords to inspect and gather information on your IT infrastructure. The following information describes how you use the qpc command line interface to complete common Quipucords tasks.
+You use the capabilities of Quipucords to inspect and gather information on your IT infrastructure. The following information describes how you use the qpc command line interface to complete common Quipucords tasks. The complete list of options for each qpc command and subcommand are listed in the qpc man page.
 
-Quipucords requires the configuration of two basic structures to manage the inspection process. A *credential* contains the access credentials, such as the username and password or SSH key of the user, with sufficient authority to run the inspection process on a particular source. A *source* defines the entity or entities to be inspected, such as a host, subnet, network, or systems management solution such as vCenter Server or Satellite. When you create a source, you also include one or more of the configured credentials to use to access the individual entities in the source during the inspection process.
+Quipucords requires the configuration of two basic structures to manage the inspection process. A *credential* contains the access credentials, such as the username and password or SSH key of the user, with sufficient authority to run the inspection process on a particular source. For more information about this authority, see `Requirements`_. A *source* defines the entity or entities to be inspected, such as a host, subnet, network, or systems management solution such as vCenter Server or Satellite. When you create a source, you also include one or more of the configured credentials to use to access the individual entities in the source during the inspection process.
```

Comment:
Is the space after `Requirements` supposed to be underlined?

Reply:
its a link in RST.

Reply:
Gotcha!